README.rst

```diff
@@ -1,6 +1,29 @@
-This is the core repository of the Open edX software. It includes the LMS
-(student-facing, delivering courseware), and Studio (course authoring)
-components.
+############
+edX Platform
+############
+| |License: AGPL v3| |Status| |Python CI|
+
+.. |License: AGPL v3| image:: https://img.shields.io/badge/License-AGPL_v3-blue.svg
+  :target: https://www.gnu.org/licenses/agpl-3.0
+
+.. |Python CI| image:: https://github.com/openedx/edx-platform/actions/workflows/ci.yml/badge.svg
+  :target: https://github.com/openedx/edx-platform/actions/workflows/ci.yml
+
+.. |Status| image:: https://img.shields.io/badge/status-maintained-31c653
+
+Purpose
+=======
```

Comment:
```suggestion
-------
```
As is, this is creating a new heading level and nesting the rest of the README under `Purpose`.